src/aiovodafone/api.py

```diff
@@ -317,6 +325,9 @@ async def get_sensor_data(self) -> dict[Any, Any]:
         data["sys_firmware_version"] = status_json["data"]["firmwareversion"]
         data["sys_hardware_version"] = status_json["data"]["hardwaretype"]
         data["sys_uptime"] = status_json["data"]["uptime"]
+        data["wan_status"] = status_json["data"]["WANStatus"]
+        data["cm_status"] = status_json["data"]["CMStatus"]
+        data["cm_mode"] = status_json["data"]["LanMode"]
```

Comment:
Another pass or review, showed that line. Why ?

```suggestion
        data["lan_mode"] = status_json["data"]["LanMode"]
```

Reply:
LanMode describes the functionality of cable modem. There are two mods available:

- router
- bridge-static

Those mods describe the functionality, which will be done by modem. Router is acting as a classical router. Bridge-static is acting a bridge device. In GUI it is named as modem mode. The decision from my side was, to take this naming. LAN mode can be misunderstood.

You can decide how the naming should be. It is your project. I am happy, with your decision.